pkg/operator/ceph/csi/template/rbd/csi-rbdplugin-provisioner-dep.yaml

```diff
@@ -46,7 +49,11 @@ spec:
             - "--csi-address=$(ADDRESS)"
             - "--v={{ .LogLevel }}"
             - "--timeout={{ .GRPCTimeout }}"
-            - "--leader-election"
+            - "--leader-election=true"
+            - "--leader-election-namespace={{ .Namespace }}"
```

Comment:
this looks duplicate we have this flag already set at line 57

Reply:
should be good now